### PR TITLE
raw images support

### PR DESCRIPTION
**Pull request by belkone**
I added support for raw base images.
To skip the logic responsible for unpacking, specify `file_target_extension` as "img" or "iso" in `remote file config`.

### CODE REVIEW

File: builder/step_extract_and_copy_image.go [hunk 1]
```diff
@@ -45,37 +45,42 @@ func (s *StepExtractAndCopyImage) Run(ctx context.Context, state multistep.State
...
-		ui.Error(fmt.Sprintf("error while unpacking %v: %s", err, out))
-		return multistep.ActionHalt
-	}
-
-	// step 4: if previously copied archive still exists, lets remove it
-	if _, err := os.Stat(dst); err == nil {
-		os.RemoveAll(dst)
+	// skip unarchive logic if provided raw image (steps: 3&4)
+	if(config.RemoteFileConfig.TargetExtension == "img" || config.RemoteFileConfig.TargetExtension == "iso") {
+        ui.Message(fmt.Sprintf("using raw image"))
```
Comment (mkaczanowski) [inline builder/step_extract_and_copy_image.go]: do you need spritnf here?

**Comment by mkaczanowski** [comment]
one minor comment, other than that it looks legit

File: builder/step_extract_and_copy_image.go (hunk 1 above)
Comment (belkone) [reply builder/step_extract_and_copy_image.go]: of course I don't need it - corrected ;)

**Comment by belkone** [comment]
I have also added relevant information to readme.
https://github.com/mkaczanowski/packer-builder-arm/blob/ba4588d472ee6fa16833762c4d9082e0f00e4343/README.md#L80-L91

**Pull request merged**